ramls/schemas/GOBIMapping.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "GOBI DataMapping Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "fieldName" : {
+      "type" : "string",
+      "description": "name of the field to map"
+    },
+    "defaultValue" : {
+      "type" : "string",
+      "description": "default value of field"
+    },
+    "fieldPath" : {
+      "type" : "string",
+      "description" : "XPath to extract data from original record"
+    },
+    "fieldType" : {
+      "enum" : [ "Integer", "Double", "String", "Lookup" ],
+      "description" : "The type to translate or convert the field value into"
+    },
+    "combinator" : {
+      "enum" : ["Multiply", "Concatenate"]
+    }
+  },
+
+  "required": [
+    "fieldName",
+    "fieldPath"
```

Comment:
not all fields are going to have an xpath expression... some data might not be present in the XML, in which case we might just specify a reasonable default.  e.g. Consider the case of "where did this order come from?" 
There's no field in the XML saying it's from GOBI; that's implied since the request was sent to mod-gobi.